docs/contributing/contributing-code/contributing-code-control-plane/running-controlplane-locally.md

```diff
@@ -26,12 +26,13 @@ If you need to manually test APIs you can reach them at the following endpoints
 
 ## Prerequisites
 
-1. Create a Kubernetes cluster, or set your current context to a cluster you want to use. The debug configuration will use your current cluster for storing data.
-2. Clone the `radius-project/radius` and `radius-project/deployment-engine` repo next to each other.
-3. Run `git submodule update --init` in the `deployment-engine` repo.
-4. Install .NET 8.0 SDK - <https://dotnet.microsoft.com/en-us/download/dotnet/8.0>.
-5. Install C# VS Code extension - <https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp>.
-6. (Optional) Configure any cloud provider credentials you want to use for developing Radius.
+1. Create a Kubernetes cluster, or set your current context to a cluster you want to use. The debug configuration will use your current cluster for storing data. 
+2. If you have access to `radius-project/deployment-engine` repo, 
```

Comment:
Can write down this and `Setup Step 4: Run Deployment Engine as docker container` as either/or? Even if someone has access, they don't need to clone it and can directly run DE in a container?